pkg/vadmin/opts/addnode/opts.go

```diff
@@ -59,3 +60,9 @@ func WithSubcluster(subcluster string) Option {
 		s.Subcluster = subcluster
 	}
 }
+
+func WithExpecteNodeNames(nodeNames []string) Option {
```

Comment:
minor: rename function
```suggestion
func WithExpectedNodeNames(nodeNames []string) Option {
```